tests/stability/test_stability.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import uuid
+from contextlib import ExitStack
+
+import coiled
+import dask
+from dask.distributed import Client
+
+SOFTWARE = f"dask-engineering/coiled_dist-py{sys.version_info[0]}{sys.version_info[1]}"
+
+
+def test_shuffle_stability():
+    with ExitStack() as stack:
```

Comment:
I'm curious why `ExitStack` is needed. Naively I would have expected us to just use 

```python
with coiled.Cluster(...) as cluster:
    with Client(cluster) as client:
        ...
```

Reply:
This is a personal preference on my part to avoid deeply nested with statements, but I don't have a strong opinion if there are objections to the pattern.

Reply:
Yeah, personally I don't know how ExitStack works, so this code would be fairly inaccessible to me (unless I go and learn things, which I probably should).  More generally, we should endeavor to [write dumb code](https://matthewrocklin.com/blog/work/2018/01/27/write-dumb-code) I think